docs/api/app.md

```diff
@@ -723,6 +723,13 @@ Returns `string` - User operating system's locale two-letter [ISO 3166](https://
 
 **Note:** When unable to detect locale country code, it returns empty string.
 
+### `app.getSystemLocale()`
+
+Returns `string` - The current system locale, fetched using Chromium's `l10n_util` library.
+Possible return values are documented [here](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/l10n/l10n_util.cc).
+
+**Note:** On Windows, you have to call it after the `ready` events gets emitted.
```

Comment:
Lets be consistent for all platforms and only allow calling this API after app ready. Also the API should throw an error if called before app ready to support this note. An example, https://github.com/electron/electron/blob/fa3cd1747556b66a3e714f99e988e47334187e44/shell/browser/api/electron_api_app.cc#L1189-L1193